packages/app/global.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@biconomy/mexa";
```

Comment:
💯

Reply:
Unrelated but we can probably get rid of this package altogether